apps-script/incident-response/JavaScript.html

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!-- [START google_chat_incident_response] -->
+<script>
+  var formDiv = document.getElementById('form');
+  var outputDiv = document.getElementById('output');
+  var clearDiv = document.getElementById('clear');
+
+  function handleFormSubmit(formObject) {
+    event.preventDefault();
+    formDiv.style.display = 'none';
+    outputDiv.innerHTML = 'Please wait while we create the space...';
+    outputDiv.style.display = 'block';
+    google.script.run
+      .withSuccessHandler(updateOutput)
+      .withFailureHandler(onFailure)
+      .createChatSpace(formObject);
+  }
+
+  function updateOutput(response) {
+    if (response.startsWith('ERROR')) {
+      outputDiv.innerHTML = '<font color="red">' + response + '</font>';
+    } else {
+      var spaceId = response.replace('spaces/', '');
+      outputDiv.innerHTML = '<p>Space created!</p><p><a href="https://mail.google.com/chat/#chat/space/' + spaceId + '" target="_blank">Open space</a></p>';
+    }
+    outputDiv.style.display = 'block';
+    clearDiv.style.display = 'block';
+  }
+
+  function onFailure(error) {
+    outputDiv.innerHTML = '<font color="red">ERROR: ' + error.message + '</font>';
```

Comment:
`<font>` tag is deprecated. use span and css instead.

Reply:
Done. Changed to a CSS class in the Div.